iabgpp-encoder/src/test/java/com/iab/gpp/encoder/GppModelTest.java

```diff
@@ -666,10 +667,10 @@ public void testConsistency() throws InvalidFieldException, EncodingException, D
     GppModel decodedModel = new GppModel(fromObjectModel.encode());
 
     Assertions.assertEquals(
-        Arrays.asList(true, true, true, true, true, true, true, true, true, true, false, false, false, false,
-            false, false, false, false, false, false, false, false, false, false),
+        new ArrayList<>(List.of(true, true, true, true, true, true, true, true, true, true, false, false, false, false,
+            false, false, false, false, false, false, false, false, false, false)),
         decodedModel.getFieldValue(TcfEuV2.NAME, TcfEuV2Field.PURPOSE_CONSENTS));
-    Assertions.assertEquals(Arrays.asList(21, 32, 81, 128, 173, 210, 238, 755),
+    Assertions.assertEquals(new ArrayList<>(List.of(21, 32, 81, 128, 173, 210, 238, 755)),
```

Comment:
Can you refrain from using List.of as its not available in JDK 8?  We should try to deal with as basic stuff as possible